libs/wingsdk/src/std/display.ts

```diff
@@ -0,0 +1,50 @@
+import { IConstruct } from "constructs";
+
+const DISPLAY_SYMBOL = Symbol.for("@winglang/sdk.std.Display");
```

Comment:
Let's merge the concepts of "display" and "connections". I think they can both be modeled through something like `std.Node`, which we should submit upstream to `constructs.Node`. They are both relevant as general purpose metadata in the construct tree for other CDKs as well.

In the meantime, we can just support this like this:

And then:

```js
std.Node.of(x).title = "hello";
std.Node.of(x).addConnection(target: x, edge: ["put()"]);
std.Node.of(x).hidden = true;
std.Node.of(x).id
std.Node.of(x).path
std.Node.of(x).addr
std.Node.of(x).addDependency(...)
```

for c in std.Node.of(x).connections {
  log(std.Node.of(c).path);
}
```

And ideally, add a keyword: `nodeof(x)` as sugar.

WDYT?

Reply:
Love the idea. I had an inkling it might be possible to combine Connections and Display under some common API but I didn't have a concrete idea at the time. But using the `nodeof` concept (that we've discussed some times in the past) sounds like a great fit. I'll try this out 👍